src/Makefile.am

```diff
@@ -26,6 +26,9 @@ include @top_srcdir@/src/Makefile.theories
 lib_LTLIBRARIES = libcvc4.la
 
 libcvc4_la_LDFLAGS = -version-info $(LIBCVC4_VERSION)
+@mk_if@eq ($(OS),Windows_NT)
+libcvc4_la_LDFLAGS += -no-undefined
```

Comment:
Yeah, I don't have an easy way to test on OS X, so see how it goes and adjust accordingly, I suppose.